tests/framework/widgets/PjaxTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace yiiunit\framework\widgets;
+
+use yii\data\ArrayDataProvider;
+use yii\widgets\ListView;
+use yii\widgets\Pjax;
+use yiiunit\TestCase;
+
+class PjaxTest extends TestCase
+{
+    public function testGeneratedIdByPjaxWidget()
+    {
+        $nonPjaxWidget1 = new ListView(['dataProvider' => new ArrayDataProvider()]);
```

Comment:
Counters should be reset here else test would fail when ran together with other tests. See latest travis build.